components/tag-list/tag-list.js

```diff
@@ -48,7 +49,7 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 	}
 
 	static get styles() {
-		return css`
+		return [super.styles, css`
```

Comment:
Not sure which is better/clearer, this or `InteractiveMixin.styles`?

Reply:
Hmmm, I get JavaScript error when trying `InteractiveMixin.styles`.

Reply:
Ah ok cool, just figured since it was `static` you could do that. No worries.